recharts/build.boot

```diff
@@ -28,6 +28,6 @@
               :requires ["react" "react-dom" "prop-types"]
               :global-exports '{recharts Recharts})
    (sift :include #{#"^cljsjs" #"^deps\.cljs$"})
+   (validate-checksums)
    (pom)
-   (jar)
-   (validate)))
+   (jar)))
```

Comment:
Please keep the `validate` task present. It will perform `validate-checksum` but also other useful checks.